_fixtures/fncall.go

```diff
@@ -142,8 +150,12 @@ func main() {
 	fn2ptrmeth := pa.PRcvr
 	var fn2nil func()
 
+	d := &Derived{}
```

Comment:
Change this to 

```
d := &Derived{ 3, Base{ 4 } }
```

I want to check that we're getting the right address for the receiver.